### PR TITLE
Feat: playwright versioning

### PR DESCRIPTION
**Pull request by andyMrtnzP**
This PR adds playwright versioning, which allows playwright clients from different versions to connect to Browserless

### CODE REVIEW

File: src/chrome-helper.ts [hunk 1]
```diff
@@ -482,6 +483,14 @@ export const convertUrlParamsToLaunchOpts = (
   const keepalive = _.isNaN(parsedKeepalive) ? undefined : parsedKeepalive;
   const parsedIgnoreDefaultArgs = parseIgnoreDefaultArgs(urlParts.query);
 
+  const playwrightVersion = (() => {
+    const uAgent = req.headers['user-agent'];
+    if (!uAgent || !uAgent.startsWith('Playwright/')) return undefined;
+
+    const split = uAgent.split('Playwright/')[1].split('.');
```
Comment (joelgriffith) [inline src/chrome-helper.ts:490]: We might be able to regex the version out a bit better than doing `.split` calls. Can you remind me an example of what the user-agent string is?

Comment (andyMrtnzP) [reply src/chrome-helper.ts:490]: Ah, I totally forgot about RegEx existence. This is the string:
`Playwright/1.25.1 (x64; windows 10.0) node/16.14`

Comment (andyMrtnzP) [reply src/chrome-helper.ts]: Used RegEx `(?<=Playwright\/)(\d+(\.\d+))`
https://regex101.com/r/y1LYDA/1

File: src/puppeteer-provider.ts [hunk 2]
```diff
@@ -377,7 +378,7 @@ export class PuppeteerProvider {
         `${jobId}: Too many concurrent and queued requests, rejecting with 429.`,
       );
       return this.server.rejectSocket({
-        header: `HTTP/1.1 429 Too Many Requests`,
```
Comment (joelgriffith) [inline src/puppeteer-provider.ts:380]: Any reason why this changed to 400 vs 429?

**Review by joelgriffith — CHANGES_REQUESTED** [review]
Two small changes, otherwise I like it!

**Review by joelgriffith — APPROVED** [review]

**Pull request merged**